pkgs/by-name/ra/rainbowcrack/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitLab
+, cmake
+, alglib
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "rainbowcrack";
+  version = "1.8-0kali1";
+
+  src = fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = "rainbowcrack";
+    rev = "kali/${version}";
+    hash = "sha256-y6jdFVQSO80YLLOitPAY8TVMUsi58B6+3bFoncwr+Qo=";
+  };
+
+  #nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ alglib ];
+
+  installPhase = ''
+    runHook preinstall
+    mkdir -p $out/{bin,usr/share/rainbowcrack}
+    cp -R *.txt rt* rcrack $out/usr/share/rainbowcrack
+    cp -R debian/helper-script/* $out/bin
+    chmod 755 $out/usr/share/rainbowcrack/rcrack
+    find $out{bin,/usr/share/rainbowcrack} -name "r*" -exec chmod 755 {} \;
+    sed -i 's/\/usr/result\/usr/g' $out/bin/*
```

Comment:
This just creates `cd result/usr/share/rainbowcrack` which won't work. You need to use `$out` here explicitly. Also there isn't really a reason why we need Kali's helper scripts at all, we can just copy binaries to `bin/` and use `wrapProgram --chdir`.